### ISSUE TITLE
DIfferent rules for different target users?

### ISSUE THREAD

**Issue by minorninth** [w3c/pronunciation#89]
Has there been any thought on the need for different pronunciation rules depending on the domain?

On one extreme, screen reader users might want extra verbose descriptions. They might prefer to sometimes err on the side of getting too much information in order to understand all of the punctuation and formatting, for example.

On the opposite extreme, there are many "assistant" features to read the screen out loud that target a much wider audience of users who aren't blind, but simply want some content read out loud. In that case, simplifying or skipping some content might be desirable.

Somewhere in-between might be users with dyslexia who want speech to augment visual reading. In that case a 1:1 correspondence might be critical, but verbose reading like for a screen reader would not be desirable.

Basically my concern is that a one-size-fits-all pronunciation rule might not work for everyone. If someone's pushed to create a pronunciation rule optimized for screen reader users, that may make things worse for another group, and vice versa.

It makes me wonder if something using CSS might be ultimately more flexible.

**Comment by alia11y** [comment]
Thank for the comment. This is good point. we previously look at the css and put a reference in our gap analysis document. We will continue to discuss this in task force further. but we agree that we need better support in adjusting pronunciation granularity.
